toolchain/lower/file_context.cpp

```diff
@@ -677,7 +681,13 @@ auto FileContext::BuildFunctionBody(SemIR::FunctionId function_id,
   };
 
   // Lower the return slot parameter.
```

Comment:
Should this comment be updated?

Unclear if this is talking about the `function.return_slot_pattern_id` sense of "return slot parameter", or if it's talking about the LLVM return slot parameter...